Docs/Lists/list.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lists | Tarang UI</title>
+    <link rel="shortcut icon" href="../../assets/tarang-favicon-2.jpeg" type="image/x-icon">
+    <link rel="stylesheet" href="../docs.css">
+</head>
+
+<body>
+
+    <div class="modal-wrapper">
+        <aside id="sidebarMain" class="tr-drawer tr-drawer-modal">
+            <div class="tr-drawer-header tr-header">
+                <h2 class="tr-drawer-heading">Components</h2>
+            </div>
+            <div class="tr-drawer-content">
+                <ul class="flex-col">
+                    <a href="../Avatars/avatar.html" class="tr-btn tr-btn-transparent tr-btn-icon-label">
+                        Avatars
+                    </a>
+                    <a href="../Alerts/alert.html" class="tr-btn tr-btn-transparent tr-btn-icon-label">
```

Comment:
What is tr ? It is confusing with transparent.

Reply:
it's the common prefix for every component class, it stands for the name of the library (Tarang), like fontawesome or bootstrap use prefixes like `.bs` or .`fa`, similarly Tarang uses `.tr` before every class to differentiate the user defined classes from library's classes